internal/store/sql_storage.go

```diff
@@ -62,6 +62,44 @@ func (s *sqlStorage) AddComments(ctx context.Context, comments ...*Comment) erro
 	return err
 }
 
+func (s *sqlStorage) QueryUsers(ctx context.Context, opts UserQueryOptions) ([]*User, error) {
+	// TODO Join and Aggregate comments table with users
+	sd := s.dialect.
+		Select(UsersID, UsersName).
+		From(UsersTable)
+
+	if opts.Limit != nil {
+		sd = sd.Limit(*opts.Limit)
+	}
+
+	// TODO Implement other options
+	// site, order, paging
```

Comment:
Are user accounts specific to a particular site? or are they shared across the whole network?

Reply:
I see that's described/related to https://linear.app/salttoday/issue/SALT-18, it looks like user accounts might be shared across multiple sites, so I'm not sure if/how we'd want to support this as a parameter